src/DetailsView/components/iframe-warning.tsx

```diff
@@ -16,8 +16,11 @@ export type IframeWarningProps = {
 export const IframeWarning = NamedFC<IframeWarningProps>('IframeWarning', props => (
     <>
         There are iframes in the target page. To have complete results,{' '}
-        <Link onClick={props.onAllowPermissionsClick}>give Accessibility Insights additional permissions</Link>; this will trigger a re-scan{' '}
-        of the test. <NewTabLink href={'https://accessibilityinsights.io/docs/en/faq'}>Learn more here.</NewTabLink>
+        <Link className="insights-link" onClick={props.onAllowPermissionsClick}>
+            give Accessibility Insights additional permissions
+        </Link>
+        ; this will trigger a rescan of the test.{' '}
```

Comment:
I'm guessing the text came from PM. I'm wondering if the phrase `"this will trigger a rescan of the test"`, specifically, using the concept of **"test"** is what we want.